### ISSUE TITLE
NTR: reduction in the number of scalp follicle

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4728]
**Preferred term label:**
Decreased number of scalp follicles

**Synonyms**
Reduction in the number of scalp follicle

**Definition (free text, please give PubMed ID)**
A reduction in the number of hair follicles in the scalp. 
PMID:28061825

'has part' some 
('has fewer parts of type' and ('inheres in' some 'hair follicle') and ('has modifier' some abnormal))

**Parent term (use hpo.jax.org/app)**
HP_0011138 'Abnormality of skin adnexa morphology'
@pnrobinson do you think this is the best parent?

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:601553
For phenopacket case report: Blanco-Kelly-2017-ABCA4

See https://github.com/TheJacksonLaboratory/LIRICAL/issues/180

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-5208-3432

**Comment by pnrobinson** [comment]
This is synonymous with Sparse hair and Hypotrichosis (both of these terms are synonyms actually and need to be merged. I will add this as a synoynm to https://hpo.jax.org/app/browse/term/HP:0004782
